CefSharp.Core/Internals/JavascriptCallbackImplFactory.h

```diff
@@ -0,0 +1,31 @@
+// Copyright � 2010-2015 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+using namespace System;
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        ref class CefSharpBrowserWrapper;
+
+        ref class JavascriptCallbackImplFactory : public IJavascriptCallbackFactory
```

Comment:
`Impl` in the name isn't consistent with the rest of the codebase. I know that causes some conflicts, so I think a name reshuffle is in order.